packages/backend/src/tests/tableManager.ts

```diff
@@ -14,7 +14,8 @@ export async function truncateTables(
     'knex_migrations',
     'knex_migrations_lock',
     'knex_migrations_backend',
-    'knex_migrations_backend_lock'
+    'knex_migrations_backend_lock',
+    'tenants' // So we do not truncate the operator tenant
```

Comment:
otherwise, we lose the operator tenant in the tenants table, and we can't insert any assets (leading to almost all tests to fail, basically).

Some alternatives:
- manually add "tenants" to list of `ignoreTables` when calling `truncateTables` in tests. We would need to do this for every test file that uses assets.
- manually run `20241205153035_seed_operator_tenant` migration in this file